.github/workflows/tests-integration.yaml

```diff
@@ -71,8 +71,6 @@ jobs:
 
       - name: run integration tests
         timeout-minutes: 10
-        env:
-          HIVE_DEBUG: 1
```

Comment:
Removing this because it causes OCLif to alter output (show stack traces) which will cause output snapshots to not match.

Reply:
Odd -- I see no reference to this env var in our codebase other than here.

Maybe @kamilkisiela remembers why this was first added
https://github.com/graphql-hive/console/pull/930/files#diff-35ee6748a6115f20e5aed37a43d75360daa460ae5c4e5a73e62f63b4f5c3be95R67